jupyter-pystata/resources/Stata.ipynb

```diff
@@ -6,29 +6,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import os\n",
```

Comment:
the current code below errors when the license file is not present, the jupyter kernel just dies and it's unclear to the user what caused it
```
import os
import shutil
if (os.path.isfile(f"/home/{os.environ['NB_USER']}/pd/stata.lic")
    and not os.path.isfile("/usr/local/stata17/stata.lic")):
    shutil.copyfile(f"/home/{os.environ['NB_USER']}/pd/stata.lic", "/usr/local/stata17/stata.lic")

import stata_setup
stata_setup.config("/usr/local/stata17", "mp")
```

could we replace it with this code with a bit more error handling?
```
import os
import shutil

if (os.path.isfile(f"/home/{os.environ['NB_USER']}/pd/stata.lic")
    and not os.path.isfile("/usr/local/stata17/stata.lic")):
    shutil.copyfile(f"/home/{os.environ['NB_USER']}/pd/stata.lic", "/usr/local/stata17/stata.lic")

if (os.path.isfile("/usr/local/stata17/stata.lic")):
    import stata_setup
    stata_setup.config("/usr/local/stata17", "mp")
else:
    print("Did not find license, please provide it at '/pd/stata.lic'")
```